Change the default font stack. Source Code Pro is GNOME's default monospaced font as of
early 2019: https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/-/issues/2
--- config.h
+++ config.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Source Code Pro, Liberation Mono, DejaVu Sans Mono, monospace:size=13:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
